tests/unit/swagger/templateHelpers.spec.ts

```diff
@@ -325,7 +325,7 @@ describe('ValidationService', () => {
   describe('Param validate', () => {
     it('should apply defaults for optional properties', () => {
       const value = undefined;
-      const propertySchema: TsoaRoute.PropertySchema = { dataType: 'integer', default: '666', required: false, validators: {} };
+      const propertySchema: TsoaRoute.PropertySchema = { dataType: 'integer', default: 666, required: false, validators: {} };
```

Comment:
Can you revert these changes? At least while we don't check/fix when creating the metadata for validation.

Reply:
If revert it, test fails... it's ok?

Reply:
The problem then is that this is the schema with the default being a string is what we would generate "in the real world".

Reply:
ah ok, so this PR can't work!